utils/wkhtmltopdf.js

```diff
@@ -1,30 +1,28 @@
-var spawn = require("child_process").spawn;
+const { spawn } = require("child_process");
 
-wkhtmltopdf = function (html, options) {
-    options = options || [];
+wkhtmltopdf = function (html, options = []) {
```

Comment:
this can be simplified to `module.exports = function (html, options = []) {`.

also, `const/let` declaration specifier is missing.